indigo_app/templates/indigo_api/_task_list.html

```diff
@@ -6,10 +6,14 @@
     <tr>
       {% block table-header %}
       <th scope="col" colspan="2">Task</th>
-      {% if place %}
-        <th scope="col">Place</th>
+      {% if not work_overview %}
```

Comment:
This breaks the concept of encapsulation, that this template should only need to know about its local state. `work_overview` implies it has broader knowledge, that it understands that when it's being used in a particular context, then certain things should happen differently.

The previous `place` indicator inverts this: the including context is telling the template to behave in a certain way. This preserves encapsulation.

Instead, I suggest you keep `place` and either just leave 'Work/Document' as it is (actually I think "Work or Document" is better), or add another variable to control the display of that column.

Reply:
I would like to not display the work when on the work overview, it feels silly to include it.
When you say 'add another variable to control the display of that column', would it be fine to just call it something else, like `dont_display_works`? I don't want to make it `display_works` for all other instances, the default should be to display them.

Reply:
something like
```
  {% if hide_works %}
    <th scope="col">Document</th>
  {% else %}
    <th scope="col">Work or Document</th>
  {% endif %}
```

Reply:
Yes, absolutely. It's exactly the same concept as 'show/don't show the place'.